fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -625,10 +690,18 @@ class FirefoxColors(
         scrim: Color = this.scrim,
         gradientStart: Color = this.gradientStart,
         gradientEnd: Color = this.gradientEnd,
```

Comment:
We should probably prefix these with `layer`. We could do this now or as a followup depending on your preference.

Reply:
Let's do this as a follow-up since the changelog will likely get polluted from the usage changes.

I've opened https://bugzilla.mozilla.org/show_bug.cgi?id=1851654